doc/button/shutdown-button.md

```diff
@@ -0,0 +1,76 @@
+# Shutdown button for donkeycar
+ 
+> 🇫🇷 **You can read this documentation in [french](doc/button/Boutton_poussoir.md)**
+
+Method of use for installing and using a push button to turn off a donkeycar
+
+> ⚠️ If you using our raspberry pi image generate by ansible and packer. All software requirements are already installed and configured on your car.
+You should just follow our [electronic shematic](doc/schema/schema-electronique.png) for install it 
+
+## Install requirements :
+
+install Wiring Pi
+```
+# apt-get install wiringpi
+```
+
+## Create script bash :
+Create `pin-shutdown.sh` file at `/root` folder
+```
+# nano /root/pin-shutdown.sh
+```
+In this file you should add those lines :
+```bash
+#!/bin/bash
+
```

Comment:
Old version, this might cause spurious shutdowns, you should update it or add a disclosure such as: "This script might cause unwanted shutdowns with some buttons registering a false positive because of the car shaking"